Sources/MapboxCoreNavigation/NativeHandlersFactory.swift

```diff
@@ -30,7 +30,9 @@ class NativeHandlersFactory {
     // MARK: - Native Handlers
     
     lazy var historyRecorder: HistoryRecorderHandle? = {
-        HistoryRecorderHandle.build(forHistoryDir: historyDirectoryURL?.path ?? "", config: configHandle)
+        historyDirectoryURL.flatMap {
```

Comment:
I found an error in the logs from NavNative. This fixes the issue.

Reply:
oh, I got used to this syntax and didn't think twice before using it. If you think that it is better to use simpler, more obvious syntax, let me know, I will update this code.